storage/migrations/mysql/5_campaign_schedules.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+
+CREATE TABLE IF NOT EXISTS `campaign_schedules`
+(
+    `id`            varbinary(27)       primary key,
+    `campaign_id`   integer unsigned    NOT NULL,
```

Comment:
I'm thinking that the `campaign_id` should be unique, since we must have only one record of a schedule.